lib/ansible/module_utils/basic.py

```diff
@@ -1409,7 +1420,7 @@ def get_bin_path(self, arg, required=False, opt_dirs=None):
 
         return bin_path
 
-    def boolean(self, arg):
+    def boolean(self, arg: t.Any) -> bool:
```

Comment:
Most of the time, it's possible to use `object` instead of a `typing.Any`. This is an official recommendation from the MyPy docs.
```suggestion
    def boolean(self, arg: object) -> bool:
```